communication/slack-config/users.yaml

```diff
@@ -7,6 +7,7 @@ users:
   AlexB138: U3JA3MDGV
   alisondy: U9CBCBLCV
   ameukam: U68KPQ448
+  annajung: U8SLB1P2Q
```

Comment:
Same here -- can you move anna down below ankeesler so it's in alphabetical order? Thanks!

Reply:
Alpha-sort strikes again!!

Reply:
Fixed in https://github.com/kubernetes/community/pull/5745.